.gitattributes

```diff
@@ -20,3 +20,7 @@
 *.PDF	 diff=astextplain
 *.rtf	 diff=astextplain
 *.RTF	 diff=astextplain
+
+sysadmin/files/referral-spam.conf diff=ansible-vault merge=binary
```

Comment:
I think it makes sense to encrypt [open-context-py/referral-spam.conf at ry20191004 · rdhyee/open-context-py](https://github.com/rdhyee/open-context-py/blob/ry20191004/sysadmin/files/referral-spam.conf) -- hence this line to help understand diffs in that file